arm-compute/2016-03-30/swagger/containerService.json

```diff
@@ -159,7 +185,7 @@
         "tags": [
           "ContainerService"
         ],
-        "operationId": "ContainerService_List",
+        "operationId": "ContainerService_ListByResourceGroup",
```

Comment:
ContainerService**s**_ListByResourceGroup
Noun_Verb ->Noun should be in plural form